examples/tutorials/run-scalable-flow-with-pipeline/pipeline.ipynb

```diff
@@ -0,0 +1,613 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Why use Azure machine learning(ML) pipelines to run your flows on the cloud?\n",
+    "In real-world scenarios, flows serve various purposes. For example, consider a flow designed to evaluate the relevance score for a communication session between humans and agents. Suppose you want to trigger this flow every night to assess today’s performance and avoid peak hours for LLM (Language Model) endpoints. In this common scenario, people often encounter the following needs:\n",
+    "- Handling Large Data Inputs: Running flows with thousands or millions of data inputs at once.\n",
+    "- Scalability and Efficiency: Requiring a scalable, efficient, and resilient platform to ensure success.\n",
+    "- Automations: Automatically triggering batch flows when upstream data is ready or at fixed intervals.\n",
+    "\n",
+    "__Azure ML pipelines__ address all these offline requirements effectively. With the integration of prompt flows and Azure ML pipeline, flow users could very easily achieve above goals and in this tutorial, you can learn:\n",
+    "- How to use python SDK to automatically convert your flow into a 'step' in Azure ML pipeline.\n",
+    "- How to feed your data into pipeline to trigger the batch flow runs.\n",
+    "- How to build other pipeline steps ahead or behind your prompt flow step. e.g. data preprocessing or result aggregation.\n",
+    "- How to setup a simple scheduler on my pipeline.\n",
+    "- How to deploy pipeline to an Azure ML batch endpoint. Then I can invoke it with new data when needed.\n",
+    "\n",
```

Comment:
shall we also revamp the examples here？ https://github.com/Azure/azureml-examples/blob/main/sdk/python/jobs/pipelines/1l_flow_in_pipeline/flow_in_pipeline.ipynb